state/applicationoffers_test.go

```diff
@@ -167,6 +167,22 @@ func (s *applicationOffersSuite) TestAddApplicationOfferBadEndpoints(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 }
 
+func (s *applicationOffersSuite) TestAddApplicationOfferContainerScopedEndpoints(c *gc.C) {
```

Comment:
nit: I would name this test more explicitly to take into account that it should fail when the endpoint is not global (e.g. `TestFaillAddApplicationOfferNonGlobalEndpoint`)

Reply:
Good point, changed